registration/registration-services/src/main/java/io/mosip/registration/dao/impl/MasterSyncDaoImpl.java

```diff
@@ -251,4 +250,9 @@ public Location getLocation(String code, String langCode) {
 	public List<LocationHierarchy> getAllLocationHierarchy(String langCode) {
 		return locationHierarchyRepository.findAllByIsActiveTrueAndLangCode(langCode);
 	}
+	
+	@Override
+	public List<LocationHierarchy> getAllLocationHierarchy() {

```

Comment:
We need to get only the count, fetching all the entities will be overhead here.

Reply:
Returned count instead of entities.